src/Payload.js

```diff
@@ -0,0 +1,33 @@
+const defaults = {
+	data: null,
+	type: null,
+}
+
+class Payload {
+	constructor({
+		data = defaults.data,
+		type = defaults.type,
+	} = defaults) {
+		Object.assign(this, {
+			data,
+			type,
+		})
+	}
+
+	static serialize = (payload) => JSON.stringify(payload)
+
+	static deserialize = (serializedPayload) => {
+		const deserializedPayload = JSON.parse(serializedPayload, (key, value) => {
+			if (typeof value === 'object') {
+				const stringifiedType = value.type || ''
+				if (stringifiedType === 'Buffer') {
+					return Buffer.from(value.data || '')
+				}
```

Comment:
If I'm reading this correctly, if the value is an object that _isn't_ a Buffer, it gets excluded from the deserialized response. Is that intentional?

I feel like we need to describe excepted Payload composition better somewhere; perhaps not in the documentation for the `Payload.deserialize` method, but _somewhere_.

Reply:
Wait, I am dumb and that is not true. It will just continue onward to L27. Sorrrry!